sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -559,17 +548,18 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
                 && podSpec.getNetworks().isEmpty()
                 && podSpec.getRLimits().isEmpty()
                 && secretVolumes.isEmpty()) {
+            // Nothing left to do.
             return containerInfo.build();
         }
 
-        boolean shouldAddImage = podSpec.getImage().isPresent() && addExtraParameters &&
+        boolean shouldAddImage =
```

Comment:
It isn't. I'd normally have dropped the variable but the explicit name gives some more context on what the boolean logic is trying to determine